tests/test_integration.py

```diff
@@ -1763,37 +1764,54 @@ def test_issues_539_abi3_resolution():
 def assert_reproducible_build(args):
     # type: (List[str]) -> None
     with temporary_dir() as td:
-        pex1 = os.path.join(td, "1.pex")
-        pex2 = os.path.join(td, "2.pex")
-
         # Note that we change the `PYTHONHASHSEED` to ensure that there are no issues resulting
         # from the random seed, such as data structures, as Tox sets this value by default. See
         # https://tox.readthedocs.io/en/latest/example/basic.html#special-handling-of-pythonhashseed.
         def create_pex(path, seed):
-            result = run_pex_command(args + ["-o", path], env=make_env(PYTHONHASHSEED=seed))
+            result = run_pex_command(
+                args + ["-o", path, "--disable-cache"], env=make_env(PYTHONHASHSEED=seed)
+            )
             result.assert_success()
 
-        create_pex(pex1, seed=111)
-        # We sleep to ensure that there is no non-reproducibility from timestamps or
-        # anything that may depend on the system time. Note that we must sleep for at least
-        # 2 seconds, because the zip format uses 2 second precision per section 4.4.6 of
-        # https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT.
-        safe_sleep(2)
-        create_pex(pex2, seed=22222)
-        # First explode the PEXes to compare file-by-file for easier debugging.
-        with ZipFile(pex1) as zf1, ZipFile(pex2) as zf2:
-            unzipped1 = os.path.join(td, "pex1")
-            unzipped2 = os.path.join(td, "pex2")
-            zf1.extractall(path=unzipped1)
-            zf2.extractall(path=unzipped2)
-            for member1, member2 in zip(sorted(zf1.namelist()), sorted(zf2.namelist())):
-                member1_path = os.path.join(unzipped1, member1)
-                if os.path.isdir(member1_path):
+        def create_multiple_pexes():
+            paths = []
+            for index in range(3):
+                path = os.path.join(td, "{}.pex".format(index))
+                paths.append(path)
+                # We sleep to ensure that there is no non-reproducibility from timestamps or
+                # anything that may depend on the system time. Note that we must sleep for at least
+                # 2 seconds, because the zip format uses 2 second precision per section 4.4.6 of
+                # https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT.
+                if index > 0:
+                    safe_sleep(2)
+                create_pex(path, seed=(index * 497) + 4)
```

Comment:
This seemed and I assume was mostly arbitrary; however, switching to just using the index masked a bug that only surfaced in Python 3.5 over in Chroot.zip now fixed in this PR.